libs/core/src/lib/fixed-card-layout/fixed-card-layout.component.ts

```diff
@@ -63,6 +75,92 @@ class CardDropped {
 
 type CardColumn = CardDefinitionDirective[];
 
+@Directive({ selector: '[fdGroup]' })
```

Comment:
Let's make selector and class names more meaningful - `fdDndGroup` or like that.

Reply:
I guess it maybe useful to have the dnd feature with keyboard support not only for fixed card layout but as utility. I'll ask about it on the team call.

Reply:
done